app/views/layouts/_sidebar.html.erb

```diff
@@ -35,12 +35,35 @@
         <% if current_user.casa_admin? || current_user.supervisor? %>
           <%= render partial: "layouts/other_duties_nav_item" %>
         <% end %>
-        <li class="<%= active_class(casa_cases_path) %> nav-item">
-          <%= link_to casa_cases_path do %>
-            <i class="lni lni-folder mr-10"></i>
-            <%= cases_index_title %>
-          <% end %>
-        </li>
+
+        <% if !policy(Volunteer).index? %>
```

Comment:
@Rafael-Martins - That's a good catch. Please leverage the user's methods role checking methods here i.e. ` <% if current_user.volunteer? %>`:
https://github.com/rubyforgood/casa/blob/8f08a6ab596a120a686025e48d7ec56f49b2bb16/app/models/user.rb#L60-L70

Reply:
yeah, I was thinking in something like this, thanks!